src/hotspot/cpu/riscv/vm_version_riscv.cpp

```diff
@@ -179,6 +179,6 @@ void VM_Version::initialize_cpu_information(void) {
   _no_of_threads = _no_of_cores;
   _no_of_sockets = _no_of_cores;
   snprintf(_cpu_name, CPU_TYPE_DESC_BUF_SIZE - 1, "RISCV64");
-  snprintf(_cpu_desc, CPU_DETAILED_DESC_BUF_SIZE, "RV64%s", _features_string);
+  snprintf(_cpu_desc, CPU_DETAILED_DESC_BUF_SIZE, "RISCV64%s", _features_string);
```

Comment:
Keep it here. It's supposed to be RV64.

Reply:
'RV64' will cause the test/jdk/jdk/jfr/event/os/TestCPUInformation.java case fail.

Reply:
Done.